dll/shellext/acppage/CLayerUIPropPage.cpp

```diff
@@ -95,10 +95,16 @@ CLayerUIPropPage::CLayerUIPropPage()
 , m_RegistryEnabledLayers(0)
 , m_EnabledLayers(0)
 {
+    CComBSTR title;
+    title.LoadString(g_hModule, IDS_COMPAT_TITLE);
+    m_psp.pszTitle = title.Detach();
+    m_psp.dwFlags |= PSP_USETITLE;
 }
 
 CLayerUIPropPage::~CLayerUIPropPage()
 {
+    CComBSTR title;
+    title.Attach((BSTR)m_psp.pszTitle);
```

Comment:
Why is the cast needed?

Reply:
Because the type of pszTitle is `LPCWSTR`, but we know that we put a BSTR in there.

Reply:
Ah ok, thanks!